src/desktop/Constraint.cpp

```diff
@@ -62,8 +63,12 @@ void CConstraint::onCommit() {
     const auto COMMITTED = m_pConstraint->current.committed;
 
     if (COMMITTED & WLR_POINTER_CONSTRAINT_V1_STATE_CURSOR_HINT) {
-        m_bHintSet      = true;
-        m_vPositionHint = {m_pConstraint->current.cursor_hint.x, m_pConstraint->current.cursor_hint.y};
+        static auto PXWLFORCESCALEZERO = CConfigValue<Hyprlang::INT>("xwayland:force_zero_scaling");
+
+        m_bHintSet       = true;
+        const auto ISXWL = m_pOwner->getWindow()->m_bIsX11;
```

Comment:
getWindow() may be null

Reply:
Fixed. Btw this is my first PR. Is it considered bad to commit to the main branch for a PR? Also, am I generally supposed to resolve the conversation or should I leave it to you?

Reply:
Is window in lower case fine?

Reply:
> Is it considered bad to commit to the main branch for a PR?

No

> Also, am I generally supposed to resolve the conversation or should I leave it to you?

For me it's whatever